Makefile

```diff
@@ -115,6 +115,17 @@ else
 	echo -e "\nIntegration tests skipped (create a '.env.integration' file to run them)"
 endif
 
+test_integration_cleanup: clean
```

Comment:
@bdero Could you please add some documentation about this to the ["Running the tests"](https://github.com/open-craft/opencraft#running-the-tests) section of the README?